.travis.yml

```
@@ -4,30 +4,24 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
   - 1.0
+  - 1.3
   - nightly
```

Add 32-bit testing since we dropped AppVeyor
```suggestion
  - nightly
arch:
  - x64
  - x86
```